app/test_main.py

```diff
@@ -1,3 +1,23 @@
 from app.main import get_human_age
+import pytest
```

Comment:
Consider adding test cases for invalid inputs, such as negative numbers or non-integer types, to ensure the function handles these gracefully. This aligns with the checklist recommendation to test edge situations and ensure the function raises the correct exceptions for invalid data types.